package/netevent/package

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(netevent)
+pkgdesc="Input-Event device cloning utility"
+url="https://github.com/Blub/netevent"
+pkgver=68e648e-1
```

Comment:
By the looks of it, the 2.1 tag is the latest version and should be used instead? It looks like you are using unreleased code? It might be best to ask for a release tag to be added. Using git hashes for a version number will cause issues with upgrades not being detected as newer. See https://github.com/toltec-dev/toltec/blob/stable/docs/package-guide.md#writing-the-recipe

Reply:
2.1 does not incorporate some recent changes that are needed to pair this with vnsee. I'll try to ask for a release tag.